locales/ta_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -102,11 +102,11 @@ msgstr ""
 
 msgctxt "frontpage"
 msgid "%sHelp Spread DuckDuckGo!%s"
-msgstr ""
+msgstr "%sடக்டக்கோவை பிரபலப்படுத்த உதவுங்கள்! %s "
 
 msgctxt "frontpage"
 msgid "%sHelp Spread DuckDuckGo%s."
-msgstr ""
+msgstr "%sடக்டக்கோவை பிரபலப்படுத்த உதவுங்கள்%s"
```

Comment:
Is it possible to leave the name of the company untranslated and written in the latin alphabet? I don't know the language, but we have automated checks that prevent us from merging strings when the brand name has been translated.